gwpy/io/kerberos.py

```diff
@@ -49,12 +47,18 @@ class KerberosError(RuntimeError):
     pass
 
 
-def kinit(username=None, password=None, realm=None, exe=None, keytab=None,
-          krb5ccname=None, verbose=None):
-    """Initialise a kerberos (krb5) ticket.
+def kinit(
+        username=None,
+        password=None,
+        realm=None,
+        exe="kinit",
```

Comment:
Just checking: unless I missed something, is it intended behaviour that the default changed from `exe=which('kinit')` to `exe='kinit'`? Those seem like two different cases for `exe`

Reply:
@eagoetz, this is a deliberate change. `subprocess` effectively does a `which` call internally anyway so I decided to remove it and let `subprocess` handle it.